portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,241 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  font-family: 'Open Sans', sans-serif;
+}
+body {
+  padding: 0;
+  margin: 0;
+}
+html {
+  scroll-behavior: smooth;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+/*--- navigation bar ---*/
+h1 {
+    background-color: #84bdff;
+    padding: 0 10px 0 10px;
+}
+.navbar {
+	background: #79b8ff;
+}
+
+.nav-link,
+.navbar-brand {
+	color: #fff;
+    font-family: 'Open Sans', sans-serif;
+	cursor: pointer;
+}
+.icon1{
+    height: 50%;
+    width: 47px;
+}
+
+.icon2{
+    height: 50%;
+    width: 30px;
+}
+
+.nav-link {
+	margin-right: 1em !important;
+}
+
+.nav-link:hover {
+	color: #000;
+}
+
+.navbar-collapse {
+	justify-content: flex-end;
+}
+
+/*--- header ---*/
+
+.header {
+	background-color: #79b8ff;
+	background-size: cover;
+	background-position: center;
+	position: relative;
+}
+
+.overlay {
+	position: absolute;
+	min-height: 100%;
+	min-width: 100%;
+	left: 0;
+	top: 0;
+	background: rgba(0, 0, 0, 0.6);
+}
+
+.description {
+	left: 50%;
+	position: absolute;
+	top: 45%;
+	transform: translate(-50%, -55%);
+	text-align: center;
+}
+
+.description h1 {
+	color: #494949;
+    font-family: 'Open Sans', sans-serif;
+
+}
+
+.description a {
+	color: #fff;
+	font-size: 1.3rem;
+    font-family: 'Open Sans', sans-serif;
+	line-height: 1.5;
+    cursor: pointer;
+    text-decoration: none;
+    font-style: italic !important;
+
+}
+
+/*--- feature section ---*/
+.section{
+    padding: 3rem 1.5rem;
+}
+.container {
+    flex-grow: 1;
+    margin: 0 auto;
+    position: relative;
+    width: auto;
+}
+.row {
+  display: flex;
+}
+
+.column {
+  flex: 50%;
+}
+.columns {
+    margin-left: -0.75rem;
+    margin-right: -0.75rem;
+    margin-top: -0.75rem;
+}
+
+.features {
+	margin: 4em auto;
+	padding: 1em;
+	position: relative;
+}
+
+.feature-title {
+	color: #333;
+	font-size: 1.3rem;
+	font-weight: 700;
+	margin-bottom: 20px;
+	text-transform: uppercase;
+}
+
+.features img {
+	-webkit-box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.4);
+	-moz-box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.4);
+	box-shadow: 1px 1px 4px rgba(0, 0, 0, 0.4);
+	margin-bottom: 16px;
+}
+
+.features .form-control,
+.features input {
+	border-radius: 0;
+}
+
+.features .btn {
+	background-color: #ffffff;
+	border: 1px solid #ffffff;
+	color: #fff;
+	margin-top: 20px;
+}
+
+.features .btn:hover {
+	background-color: #333;
+	border: 1px solid #333;
+}
+
+#fact-container {
+  margin-top: 20px;
+}
+
+.project-area {
+	background: #79b8ff;
+}
+.project-title{
+    color: #333;
+	font-size: 1.6rem;
+	font-weight: 700;
+	text-transform: uppercase;
+    margin-left: 700px;
+}
+
+.project-name{
+    color: #333;
+	font-size: 1.3rem;
+	font-weight: 700;
+	text-transform: uppercase;
+    margin-left: 700px;
+    font-style: italic !important;
+
```

Comment:
nit: more tabs (instead of spaces), + unnecessary linebreak, here and elsewhere.